src/components/Workspace.jsx

```diff
@@ -331,55 +352,33 @@ class Workspace extends React.Component {
     this.props.dispatch(editorFocusedRequestedLine());
   }
 
-  async _handleExportGist() {
-    if (this.props.clients.gists.exportInProgress) {
-      return;
-    }
+  _handleExportGist() {
+    this._gistExportWindow = openWindowWithWorkaroundForChromeClosingBug(
+      `data:text/html;base64,${spinnerPage}`,
+    );
 
-    if (!this.props.currentUser.authenticated) {
-      // eslint-disable-next-line no-alert
-      if (!confirm(t('workspace.confirmations.anonymous-gist-export'))) {
```

Comment:
Deleting it was intentional, yep—figure it’s not really helping anything, and blocking alerts are generally a high UX cost. Good call on the translation!